app/src/main/java/hackathon/tastemydroid/ui/fragments/RecipeListFragment.kt

```diff
@@ -21,7 +21,7 @@ class RecipeListFragment : Fragment(R.layout.fragment_recipe_list) {
     }
 
     private fun initRecycler() {
-        binding.recycler.adapter = ProductsListAdapter()
+        binding.recycler.adapter = ProductsListAdapter(emptyList())
```

Comment:
should be  RecipeListAdapter. I fixed it in https://github.com/SigarevA/TasteMyDroid/pull/8/commits/7fca405331665e9233c8956c0eeb873e7ae8b39b